web/styles/popovers.css

```diff
@@ -337,24 +337,42 @@ ul.popover-group-menu-member-list {
     flex-direction: column;
 
     .simplebar-content {
+        display: grid;
+        grid-template-columns:
+            var(--user-group-popover-horizontal-padding)
+            [group-member-icon-start] minmax(5px, auto)
+            [group-member-icon-end] var(--user-group-popover-icon-text-gap)
+            [group-member-name-start] 1fr [group-member-name-end] var(
+                --user-group-popover-horizontal-padding
+            );
         width: unset;
     }
+
+    .popover-group-menu-member {
+        display: grid;
+        grid-template-columns: subgrid;
+        grid-column: 1 / -1;
+        align-items: center;
+    }
+
+    .popover-group-member-icon {
+        justify-self: center;
+        grid-column: group-member-icon-start / group-member-icon-end;
```

Comment:
You should be able to just reference `group-member-icon` here, and below. That's the magic of the -start and -end on inline references. Though again, I think this would read better with `grid-template-areas:`

Reply:
Fixed, as we now just use `grid-area: group-member-icon` and `grid-area: group-member-name`.